src/binder/binder.cpp

```diff
@@ -245,6 +245,10 @@ function::TableFunction Binder::getScanFunction(FileType fileType, const ReaderC
             csvConfig.parallel ? ParallelCSVScan::name : SerialCSVScan::name, inputTypes,
             functions);
     } break;
+    case FileType::JSON: {
```

Comment:
This doesn't look right to me, json is an extension type and should not be known by the core(inlcuding binder).

Reply:
Then how should the core invoke the JSON scan function?